web/index.html

```diff
@@ -42,14 +42,274 @@
                 </div>
                 <div class="mt1">
                   <label class="fl fw4 lh-copy f5 near-white pb2" for="email">Email address</label>
-                  <input class="pa2 input-reset bn br2 bg-white w-100" type="email" name="email" placeholder="you@example.com" id="email">
+                  <input class="pa2 input-reset bn br2 bg-white w-100" type="email" name="email" placeholder="you@example.com" id="email" required>
+                </div>
+                <div class="mt1">
+                  <label class="fl fw4 lh-copy f5 near-white pb2" for="country">Country</label>
+                  <select class="pa2 input-reset bn br2 bg-white w-100" name="country" placeholder="Select a Country..." id="country" size="10" required>
+					<option value="">Select a Country...</option>
+					<option value="Afganistan">Afghanistan</option>
```

Comment:
Seriously? Why Afganistan is the first country in the list? What are the chances our StackStorm / Slack users are from Afganistan????